### DISCUSSION TITLE
Opt out of SMS 2FA fallback (two_factor_authentication.sign_in_fallback_sms)

### DISCUSSION THREAD

**Discussion by travispaul** [community/community#22948] [New to GitHub]
<p>I have 2FA enabled, I have my recovery codes printed and physically backed up in multiple locations. I would like to opt out of SMS 2FA fallback due to the potential for SMS intercept and SIM swapping/hijacking.</p>
<p>There seems to be no way to disable SMS recovery in security settings, if there is a way, please advise.</p>
<p>Thanks</p>

**Comment by travispaul** [comment, marked as answer]
<p>I figured it out, within your security settings, if you click “Edit” next to “Fallback SMS number”, and scroll down the page, there is an option to “clear” the phone number.</p>

**Comment by seacaet1** [comment]
<p>Could you give me a ring [edited by moderator] stanford</p>

**Comment by bluelou** [comment]
<p>I don’t see any option to ‘clear’ my primary SMS from 2FA (secondary SMSalready removed).  I’d like to remove the primary SMS.  Any other suggestions?</p>

**Comment by minherz** [comment]
According to [official documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/changing-your-two-factor-authentication-method) if there is more than one MFA method, I can remove one by clicking on the menu and selecting "Remove" instead of "Edit".

I have two MFA methods set up: SMA and Security Keys. The preferred method is set to Security Keys. However, I see only "Edit" option in front of SMS method. There is no way to remove SMS method for me.

**Comment by ghost** [comment]
I went as far as trying to get and register a Yubikey, and have been unable to remove the SMS 2FA.  I recently witnessed an attack by SMS hijack, so I'm upset that I physically cannot seem to do anything to disable this and that I don't have authority to remove a ****ing phone number from my own account.

**Comment by jpinnix** [comment]
I know this was answered, but it didn't mention this other potential issue.

Apparently the GitHub Mobile app doesn't qualify as one of the two other methods they require, so you must have both a security key and an authentication app, then you will see Remove as an option for SMS.
